SnapshotTests/SnapshotTestCase.swift

```diff
@@ -189,3 +189,38 @@ extension Snapshotting where Value == UIView, Format == UIImage {
         )
     }
 }
+
+extension Snapshotting where Value == UIViewController, Format == UIImage {
+    static var extra3LargeFontStrategyLandscape: Self {
+        let traits = UITraitCollection(traitsFrom: [
+            .init(forceTouchCapability: .available),
```

Comment:
We don't as far as I know. Those traits were taken from the other available iPhone models, because unfortunately iPhone 13 is not available. Perhaps this allows some other kind of testing and those traits are just bundled together for easier use.